lib/Twig/Cache/Filesystem.php

```diff
@@ -73,6 +78,15 @@ public function write($key, $content)
         if (false !== @file_put_contents($tmpFile, $content) && @rename($tmpFile, $key)) {
             @chmod($key, 0666 & ~umask());
 
+            if (self::FORCE_BYTECODE_INVALIDATION == ($this->options & self::FORCE_BYTECODE_INVALIDATION)) {
```

Comment:
should be ``===`` to respect coding standards (using strict comparison everywhere it is possible rather than letting PHP convert types)